modules/microshift-manifests-override-paths.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-applications.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-manifests-override-paths_{context}"]
+= Override the list of manifest paths
+You can override the list of default paths with the following procedure.
+
+.Procedure
+
+. In the configuration file, set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++`.
++
+[source,terminal]
+----
+manifests:
+    kustomizePaths:
+        - "/opt/alternate/path"
+----
+
+. The values of `kustomizePaths` may be glob patterns.
++
+[source,terminal]
+----
+manifests:
+    kustomizePaths:
+        - "/opt/alternative/path.d/*"
+----
+
+. To disable loading manifests, set the configuration option to an empty list.
```

Comment:
Somewhere near here we should have a note/warning that the configuration file overrides the defaults entirely, so if the kustomizePaths value is set to _anything_, only the values in the configuration file are used. That's why setting the value to an empty list disables manifest loading.